### PR TITLE
StateManager: Pass intermediates using normalized values, rename coerce to normalize

### PR DESCRIPTION
**Pull request by paulmelnikow**

### CODE REVIEW

**Comment by jbeard4** [comment]
I used functools.wrap, and I think I'm using it the right way. Requires this small change to artifax: https://github.com/curvewise-forks/artifax/pull/3

I am now getting this error:

```
    @classmethod
    def coerce(cls, value: t.Any) -> MyModel:
        if not isinstance(value, MyModel):
>           raise ValueError(f"Can't coerce {type(value).__name__} to {cls.__name__}")
E           ValueError: Can't coerce str to MyModelType

werkit/compute/graph/testing_examples.py:75: ValueError
```

@paulmelnikow I wanted to discuss it with you, because I fell like this is close, but I don't fully understand the error.

**Comment by jbeard4** [comment]
Somehow we are ending up in here: https://github.com/metabolize/werkit/blob/main/werkit/compute/graph/testing_examples.py#L75

File: werkit/compute/graph/_state_manager.py [hunk 1]
```diff
@@ -47,18 +48,26 @@ def evaluate(
...
-                name: self.dependency_graph.compute_nodes[name].bind(self.instance)
-                for name in self.dependency_graph.compute_nodes.keys()
+                name: wrap_node(name, node)
+                for name, node in self.dependency_graph.compute_nodes.items()
             }
         )
         if self.store:
             afx.set(**self.store)
         afx.build(targets=targets)
-        # TODO: `afx.build()` should always return an object.
```
Comment (paulmelnikow) [inline werkit/compute/graph/_state_manager.py:59]: We're using an Artifax implementation detail to get the result so this todo (for the artifax API) is still relevant.

File: werkit/orchestrator/orchestrator_lambda/parallel.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from harrison import Timer
 
 
-lambda_client = boto3.client("lambda")
+# lambda_client = boto3.client("lambda")
```
Comment (paulmelnikow) [inline werkit/orchestrator/orchestrator_lambda/parallel.py:9]: Can you revert this (or fix it)?

Comment (jbeard4) [reply werkit/orchestrator/orchestrator_lambda/parallel.py]: Fixed in e89c1cf2a3361a00c8d64177ecbb29b8b8555ff9

**Comment by jbeard4** [comment]
@paulmelnikow This is ready for re-review.

**Comment by jbeard4** [comment]
@paulmelnikow I'm having trouble understanding why `main` is failing. Is it failing because of coverage?

**Comment by paulmelnikow** [comment]
Is it because it needs the artifax change?

**Comment by jbeard4** [comment]
@paulmelnikow This is ready for re-review.

**Comment by paulmelnikow** [comment]
Let's update the main branch in our fork from the upstream repo, and then PR the change in artifax onto that and re-tag. Also let's give this a commit message that describes what's changing. Apart from that this looks great.

**Review by jbeard4 — APPROVED** [review]

**Pull request merged**